test/functional/operation_promises_example_tests.js

```diff
@@ -2822,10 +2822,10 @@ exports.shouldCorrectlyLogoutFromTheDatabaseWithPromises = {
 
               db.close();
               test.done();
-            }).catch(function(err) {  })
-          }).catch(function(err) {  })
-        }).catch(function(err) {  })
-      }).catch(function(err) {  })
+            }).catch(function(err) {  console.dir(err )})
```

Comment:
nitpick: the only reason we have a catch for every one of these promises is because the test isn't properly chaining promises (returning them within each `then`). We could reduce the `catch` with `console.dir` to a single instance if that was done